job_groups/staging_projects.yaml

```
@@ -68,9 +68,13 @@ scenarios:
       - gnome:
           machine: 64bit
```

So that we don't need to alter the QEMURAM settings, I suggest:

```suggestion
          machine: 64bit-4g
```

---

Good suggestion! Let me revise my code.